src/main/java/ToDoTask.java

```diff
@@ -0,0 +1,11 @@
+public class ToDoTask extends Task {
```

Comment:
Since ToDo is-a task, would a name like ToDo be sufficient?

Reply:
I feel that having the name of Task would better identify the class for other developers since they may not go deep to see the inheritance from Task class.